chatcommands.py

```diff
@@ -746,15 +746,18 @@ def stopflagging():
 
 
 # noinspection PyIncorrectDocstring,PyProtectedMember
-@command(str, whole_msg=True, privileged=True)
-def standby(msg, location_search):
+@command(str, whole_msg=True, privileged=True, aliases=["standby-except"], give_name=True)
+def standby(msg, location_search, alias_used="standby"):
     """
     Forces a system exit with exit code = 7
     :param msg:
     :param location_search:
     :return: None
     """
-    if location_search.lower() in GlobalVars.location.lower():
+    reverse_search = "except" in alias_used
+
+    # Caret `^` is xor
+    if location_search.lower() in GlobalVars.location.lower() ^ reverse_search:
```

Comment:
What's the logic precedence here? `(location_search.lower() in GlobalVars.location.lower()) ^ reverse_search`, or `location_search.lower() in (GlobalVars.location.lower() ^ reverse_search)`?

Reply:
Meh, I forgo that Python chains logical operators arbitrarily, so it's actually `(A in B) and (B in C)`.